script.js

```diff
@@ -0,0 +1,47 @@
+const removeBtn = document.createElement('button');
+const addbtn = document.querySelector('.book__add-btn');
+const bookTitle = document.querySelector('.book__title');
+const bookOwner = document.querySelector('.book__owner');
+const book = document.querySelector('.book');
+const bookAuthor = document.getElementsByClassName('book__author');
+const bookName = document.getElementsByClassName('book__name');
+const bookItem = document.querySelector('.book__item');
+
+const bookList = [];
+const bookDisplay = [];
+
+addbtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const bookInfo = {
+    title: bookTitle.value,
+    author: bookOwner.value,
+  };
+
```

Comment:
- [ ] [optional] I would suggest clearing the form fields after adding a book to the collection, It will improve the user experience.